### PR TITLE
[11.0] [ADD] new module mgmsystem_nonconformity_product

### PR DESCRIPTION
**Pull request by mymage**
Ref. #263

### CODE REVIEW

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
:+1: 

Only thing is the module name, following the logic discussed in the other PRs: `mgmtsystem_nonconformity_product`

File: mgmtsystem_extended_product/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This is a bridge module between Management System and Product
+
+This module contains some new features for Management System modules.
+
+Nonconformity (NC)
+
+- Product: add a field for link a specific product.
```
Comment (ivantodorovich) [inline mgmtsystem_extended_product/readme/DESCRIPTION.rst]:
```suggestion
- Product: add a field to link a specific product.
```

File: mgmtsystem_extended_product/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* add filter on Immediate Actions section in Nonconformity form view
+* manage Template Action on statistic analysis
```
Comment (ivantodorovich) [inline mgmtsystem_extended_product/readme/ROADMAP.rst]: Can you clearify the roadmap?
TBH I'm not sure I understand it ^^

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: mgmtsystem_extended_product/readme/ROADMAP.rst (hunk 2 above)
Comment (mymage) [reply mgmtsystem_extended_product/readme/ROADMAP.rst]: Sorry, it is a typo from the original module.

File: mgmtsystem_extended_product/readme/DESCRIPTION.rst (hunk 1 above)
Comment (mymage) [reply mgmtsystem_extended_product/readme/DESCRIPTION.rst]: ok

**Review by ivantodorovich — APPROVED** [review]
same as others, just the module name on readme and manifest :)

otherwise LGTM :+1: 

^^

Thanks so much for the contributions!

**Comment by mymage** [comment]
Just an icon restyle :-)

**Review by marcelofrare — APPROVED** [review]
functionally it is ok for me

**Pull request merged**